File: compiler/testData/loadJava/compiledJava/modality/ModalityOfFakeOverrides.java
// FULL_JDK
// SKIP_IN_RUNTIME_TEST
package test;

import org.jetbrains.annotations.NotNull;
import java.util.AbstractList;

public class ModalityOfFakeOverrides extends AbstractList<String> {
    @Override
    @NotNull
    public String get(int index) {
        return "";
    }

    @Override
    public int size() {
        return 0;
    }
}

File: compiler/testData/loadJava/compiledKotlin/fromLoadJava/modality/ModalityOfFakeOverrides.kt
// FULL_JDK
// SKIP_IN_RUNTIME_TEST
package test

import java.util.AbstractList

public open class ModalityOfFakeOverrides : AbstractList<String>() {
    override fun get(index: Int): String {
        return ""
    }

    override val size: Int get() = 0
}
